testapps/RazorToolingTestApp/project.json

```diff
@@ -0,0 +1,48 @@
+﻿{
+  "dependencies": {
+    "Microsoft.AspNetCore.Diagnostics": "1.1.0-*",
+    "Microsoft.AspNetCore.Mvc": "1.1.0-*",
+    "Microsoft.AspNetCore.Razor.Design": {
+      "type": "build",
+      "version": "1.0.0-*",
+      "target": "package"
+    },
+    "Microsoft.AspNetCore.Server.IISIntegration": "1.1.0-*",
+    "Microsoft.AspNetCore.Server.Kestrel": "1.1.0-*",
+    "Microsoft.Extensions.Logging.Console": "1.1.0-*",
+    "RazorToolingTestApp.Library": "1.0.0-*"
+  },
+
+  "tools": {
+    "Microsoft.AspNetCore.Server.IISIntegration.Tools": "1.0.0-*",
+    "Microsoft.AspNetCore.Razor.Tools": "1.0.0-*"
+  },
+
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "version": "1.0.0-*",
+          "type": "platform"
+        }
+      }
+    },
+    "net451": {},
+    "net46": {},
```

Comment:
This is going to be problematic for Linux CI. 
```
/home/travis/build/aspnet/RazorTooling/testapps/RazorToolingTestApp/project.json(31,14): 
error DOTNET1011: Framework not installed: .NETFramework,Version=v4.6 
```

Does building against net46 and net462 cover important scenarios or can we just stick to the net451 baseline?